main.c

```diff
@@ -289,7 +312,15 @@ static void handle_xdg_output_description(void *data, struct zxdg_output_v1 *out
 }
 
 static void handle_xdg_output_done(void *data, struct zxdg_output_v1 *output) {
```

Comment:
`xdg_output.done` is deprecated

Reply:
https://github.com/swaywm/swaylock/pull/155/commits/853f292af0f6e1d7c675db8fff0ec2c49a05ffca moves the logic out of the xdg_output.done handler, and https://github.com/swaywm/swaylock/pull/155/commits/01a48ffff74eefc76dd15080df58b337f2d32e04 does synchronization using wl_display.sync instead of the ad-hoc events_pending counter.